File: epi_judge_cpp/count_inversions.cc
#include <vector>

#include "test_framework/generic_test.h"
using std::vector;

int CountInversions(vector<int> A) {
  // TODO - you fill in here.
  return 0;
}

int main(int argc, char* argv[]) {
  std::vector<std::string> args{argv + 1, argv + argc};
  std::vector<std::string> param_names{"A"};
  return GenericTestMain(args, "count_inversions.cc", "count_inversions.tsv",
                         &CountInversions, DefaultComparator{}, param_names);
}

File: epi_judge_python/count_inversions.py
from typing import List

from test_framework import generic_test


def count_inversions(A: List[int]) -> int:
    # TODO - you fill in here.
    return 0


if __name__ == '__main__':
    exit(
        generic_test.generic_test_main('count_inversions.py',
                                       'count_inversions.tsv',
                                       count_inversions))
